app/lib/account/models.dart

```diff
@@ -234,6 +234,22 @@ class Consent extends db.Model {
   bool shouldNotify() =>
       notificationCount == 0 ||
       DateTime.now().toUtc().isAfter(nextNotification);
+
+  /// Returns a new [Consent] object with a new parent.
+  /// Should be used only for merging users.
+  Consent changeParentUserId(String userId) {
```

Comment:
Can't we make this a utility method in `user_merger.dart`. Once we're done with `user_merger.dart` it might be better to delete it, rather than maintain it?

Reply:
What happens if we add a new field to `Consent`? Chances are that we forget to update it in `user_merger`. I'd rather keep it with the class, and if we are done with `user_merger`, we shall (eventually) delete these too.